.github/workflows/issue-context-assignment.yml

```
@@ -74,7 +74,7 @@ jobs:
 
       - name: Queue assignment and ping mentors
         if: steps.cmd.outputs.is_assign == 'true' && steps.validate.outputs.eligible == 'true'
-        uses: actions/github-script@v8
+        uses: actions/github-scriptv9.0.0
```

actionlint (1.7.12)

[error] specifying action "actions/github-scriptv9.0.0" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)